doc/manual/15-system_requirements.en.md

```diff
@@ -19,7 +19,8 @@
 
 > **NOTE:**
 > Support for Python 2.6 and 3.5 are discontinued.
-> Support for Python 2.7 is deprecated and will be discontinued in future releases.
+> Support for Python 2.7 is limited to Barman 3.4.X version and will receive only bugfixes. It will be discontinued in a near future.
```

Comment:
Should probably read "in the near future".

Reply:
thank you